src/sentry/models/useremail.py

```diff
@@ -18,6 +18,7 @@ class UserEmail(Model):
 
     user = FlexibleForeignKey(settings.AUTH_USER_MODEL, related_name='emails')
     email = models.EmailField(_('email address'))
+    global_email = FlexibleForeignKey('sentry.Email', related_name='useremails', null=True)
```

Comment:
Probably don't need this since you can just index the email string and query on it